conjurapi/authn/api_key_authenticator_test.go

```diff
@@ -2,12 +2,13 @@ package authn
 
 import (
 	"fmt"
-	. "github.com/smartystreets/goconvey/convey"
 	"testing"
+
+	"github.com/stretchr/testify/assert"
 )
 
 func TestAPIKeyAuthenticator_RefreshToken(t *testing.T) {
-	Convey("Given valid credentials", t, func() {
+	t.Run("Given valid credentials", func(t *testing.T) {
 		Login := "valid-login"
```

Comment:
I know this isn't part of your change, but I think these variables should be un-exported, unless I'm missing something:

- Login (should be `login`)
- APIKey (should be `apiKey`)
- Authenticate (should be `authenticate`)